src/main/java/com/marklogic/appdeployer/AppConfig.java

```diff
@@ -87,8 +92,8 @@ public class AppConfig {
 	// Connection info for using the client REST API - e.g. to load modules
 	private DatabaseClient.ConnectionType restConnectionType;
 	private SecurityContextType restSecurityContextType = SecurityContextType.DIGEST;
-	private String restAdminUsername = DEFAULT_USERNAME;
-	private String restAdminPassword = DEFAULT_PASSWORD;
+	private String restAdminUsername;
```

Comment:
Removing these defaults seems sensible now that all authentication strategies are available. This will only cause a break if a user is never passing in "mlUsername" and "mlPassword" in an ml-gradle app, which we now consider a bug since we don't want to assume the use of a default username/password.